src/graphql/resolvers/getCompanyNamesResolver.js

```diff
@@ -7,42 +7,18 @@ import { TABLES } from "../environment";
 
 const GetCompanyNamesResolver = async db => {
   const params = {
-    TableName: TABLES.Offer
+    TableName: TABLES.Company
```

Comment:
Do we actually need this resolver? The getCompanies resolver already can grab company names